net-misc/icaclient/icaclient-24.2.0.65.ebuild

```diff
@@ -0,0 +1,309 @@
+
+LICENSE="icaclient"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+IUSE="l10n_de l10n_es l10n_fr l10n_ja l10n_zh-CN hdx usb"
+RESTRICT="mirror strip fetch"
+
+ICAROOT="/opt/Citrix/ICAClient"
+
+QA_PREBUILT="${ICAROOT#/}/*"
+
+# we have binaries for two conflicting kerberos implementations
+# https://bugs.gentoo.org/792090
+# https://bugs.gentoo.org/775995
+REQUIRES_EXCLUDE="
+	libgssapi.so.3
+	libgssapi_krb5.so.2 libkrb5.so.3
+"
+# when using media-plugins/hdx-realtime-media-engine we better not ignore that
+REQUIRES_EXCLUDE="${REQUIRES_EXCLUDE}
+	!hdx? ( libunwind.so.1 )
+"
+# we have binaries which wouls still support gstreamer:0.10
+REQUIRES_EXCLUDE="${REQUIRES_EXCLUDE}
+	libgstapp-0.10.so.0
+	libgstbase-0.10.so.0
+	libgstinterfaces-0.10.so.0
+	libgstpbutils-0.10.so.0
+	libgstreamer-0.10.so.0
+"
+
+# video background blurring, optional
+REQUIRES_EXCLUDE="${REQUIRES_EXCLUDE}
+	libopencv_core.so.407
+	libopencv_imgcodecs.so.407
+	libopencv_imgproc.so.407
+"
+
+BDEPEND="
+	hdx? ( media-plugins/hdx-realtime-media-engine )
+"
+
+RDEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	app-crypt/libsecret
+	dev-libs/glib:2
+	dev-libs/libxml2
+	media-fonts/font-adobe-100dpi
+	media-fonts/font-cursor-misc
+	media-fonts/font-misc-ethiopic
+	media-fonts/font-misc-misc
+	media-fonts/font-xfree86-type1
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/gst-plugins-base:1.0
+	media-libs/gstreamer:1.0
+	media-libs/libogg
+	media-libs/libpng
+	media-libs/libpulse
+	media-libs/libvorbis
+	media-libs/mesa
+	media-libs/speex
+	media-libs/speexdsp
+	net-libs/libsoup:2.4
+	net-libs/webkit-gtk:4.1
+	sys-apps/util-linux
+	sys-libs/libcxx
+	sys-libs/libcxxabi
+	sys-libs/zlib
+	virtual/krb5
+	virtual/libudev
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/gtk+:3
+	x11-libs/libX11
+	x11-libs/libXaw
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXinerama
+	x11-libs/libXmu
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/pango
+	${BDEPEND}
+	!hdx? ( !media-plugins/hdx-realtime-media-engine )
+	usb? ( virtual/libudev )
+"
+
+pkg_nofetch() {
+	elog "Download the client file ${A} from
+	https://www.citrix.com/downloads/workspace-app/"
+	elog "and place it into your DISTDIR directory."
+}
+
+pkg_setup() {
+	case ${ARCH} in
+		amd64)
+			ICAARCH=linuxx64
+		;;
+		x86)
+			ICAARCH=linuxx86
+		;;
+		*)
+			eerror "Given architecture is not supported by Citrix."
+		;;
+	esac
+}
+
+src_unpack() {
+	default
+	mv "${WORKDIR}/${ICAARCH}/${ICAARCH}.cor" "${S}" || die
+}
+
+src_prepare() {
+	default
+	rm lib/UIDialogLibWebKit.so || die
+
+	cp nls/en/module.ini . || die
+	if use usb; then
+		# inspired by debian usb support package postinst
+		sed -i -e 's/^[ \t]*VirtualDriver[ \t]*=.*$/&, GenericUSB/' module.ini || die
+		sed -i -e '/\[ICA 3.0\]/a\GenericUSB=on' module.ini || true
+		echo "[GenericUSB]" >> module.ini || true
+		echo "DriverName=VDGUSB.DLL" >> module.ini
```

Comment:
I'm not sure about this, but is this correct? The `|| true` looks wrong to me.
I guess the `sed` would need a `|| die`  and the second `|| true` could be removed?

Reply:
I'm not the official maintainer of the ebuild. 
My bump is naive copying of earlier work. I don't want to change this without understanding of why it was done .
@Amynka